### DISCUSSION TITLE
Can't log back into account

### DISCUSSION THREAD

**Discussion by Nepp0** [ppy/osu#16569] [Q&A] [answered]
![osu_2022-01-22_22-44-19](https://user-images.githubusercontent.com/98234215/150652796-f004901e-df07-4b78-b817-ac8da1b4c020.jpg)

it keeps giving me this error after disconnecting me from multi and i can't log back into my account.
I've tried logging in with the stable client and it had no problem, the osu website also works fine.

**Comment by bdach** [comment, marked as answer]
i'm not sure why the website would work but not the client. are you sure the website works? and are you sure there are no firewall rules in place that may be blocking connections?

**Reply by Nepp0** [reply]
yes the website works perfectly fine. I've disabled firewall and restarted everything including the router but it keeps giving me this error when i try to log in.
also please note that this happened while i was in the middle of a map in a multi lobby. it stuttered a little bit then it took me back to title screen.

**Reply by peppy** [reply]
seems like an issue with your network

**Reply by Nepp0** [reply]
it seems like i could log into my account after using a vpn out of curiosity...
i guess that works
